downloader/src/funTest/kotlin/GitDownloadTest.kt

```diff
@@ -98,8 +100,8 @@ class GitDownloadTest : StringSpec() {
 
         "Git can download based on a version".config(tags = setOf(ExpensiveTag)) {
             val pkg = Package.EMPTY.copy(
-                id = Identifier("Test:::$REPO_VERSION"),
-                vcsProcessed = VcsInfo(VcsType.GIT, REPO_URL, "")
+                id = Identifier("Test:::$PKG_VERSION"),
+                vcsProcessed = VcsInfo(VcsType.GIT, REPO_URL, "dummy")
```

Comment:
As you do it I assume it is important to have the retries covered which leads me to the question
if we should make this more obvious or rather explicit in the test. (Reading the test it is no obvious to me that it actually is about trying multiple revisions).

Would it make sense instead of adding "dummy" to two test cases revert that part in favor of a separate additional test case using the `dummy` revision and a test name speaking about the tries?

Reply:
Would you be fine with adding comments instead that explain why "dummy" is used instead of an empty string?

Reply:
yes

Reply:
Done.

Reply:
Thanks!